msteams-platform/concepts/build-and-test/share-in-meeting.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Share in Meeting
-description: Learn how to add the share in meeting button, which allows users to share any document or third-party app to the meeting stage.
+description: Learn how to add the share to Teams button that allows users to share any document or third-party app to the meeting stage.
```

Comment:
We should not be updating the Share-to-meetings button control.

Reply:
Agreed. Was added with the initial understanding. Reverted other changes. Thanks!